src/public/views/changelist.njk

```diff
@@ -16,7 +16,16 @@
     <button class="btn btn-primary">Add {{ metadata.name | lower }}</button>
   </a>
 </div>
+
 <div class="admin-content">
+  {% if adminEntity.searchFields %}
+  <form method="GET">
+    <div class="search-bar">
```

Comment:
The form should be inside the "search-bar", that would allow us to have other stuff on the right fo the bar